Neos.Neos/Classes/View/FusionView.php

```diff
@@ -175,11 +179,20 @@ protected function getClosestDocumentNode(Node $node): ?Node
      */
     protected function getCurrentSiteNode(): Node
     {
-        $currentNode = $this->variables['site'] ?? null;
-        if (!$currentNode instanceof Node) {
-            throw new Exception('FusionView needs a variable \'site\' set with a Node object.', 1538996432);
+        $currentSiteNode = $this->variables['site'] ?? null;
+        if (!$currentSiteNode instanceof Node) {
```

Comment:
So previously this case failed and now we try to fetch the site?
Wouldn't this disguise potential bugs? And is that related to the performance optimization?